sql-prepare-plan-cache.md

```diff
@@ -119,3 +119,119 @@ MySQL [test]> select @@last_plan_from_cache;
 +------------------------+
 1 row in set (0.00 sec)
 ```
+
+## Manually clear execution plan cache
```

Comment:
@qw4990 Should we remove 'manually' since there is no where describing 'automatically clear xxx'? Seems this info is unnecessary.